resources/views/account/edit.blade.php

```diff
@@ -48,6 +48,17 @@
                     <span class="help-block">Please use a high resolution image, as it will be provided to conference organizers.</span>
                     {!! Form::file('profile_picture', null, ['class' => 'form-control']) !!}
                 </div>
+
+                <div class="form-group">
+                    {!! Form::label('enable_profile', 'Enable email notifications?', ['class' => 'control-label']) !!}<br>
+                    <span class="help-block">Do you want to receive email notifications for open CFPs?</span>
+                    <label class="radio-inline">
```

Comment:
Don't we have to set one of these to be checked by default?

Reply:
I don't think so since we have set the field in the db to false by default.